### ISSUE TITLE
c Sharp issue

### ISSUE THREAD

**Issue by StephenBrooks220413** [OmniSharp/omnisharp-vscode#3489]
Please paste the output from your clipboard

Could not find task build

**Comment by gregg-miskelly** [comment]
@StephenBrooks220413 you need to include more details if you would like help. Please describe your repro steps, and what problem you are seeing. Make sure to include the type of project that you have (example: .NET Core 3.1 web app) and your OmniSharp log.

**Comment by StephenBrooks220413** [comment]
> @StephenBrooks220413 you need to include more details if you would like help. Please describe your repro steps, and what problem you are seeing. Make sure to include the type of project that you have (example: .NET Core 3.1 web app) and your OmniSharp log.

I had experienced problems with trying to run C Sharp in visual studio. I went through the extensions and installed anything with .Net and c sharp to help but nothing did. So I uninstalled visual studio and reinstalled it with the options to add it during installation. That solved the problem. The program wouldn't run because it was missing the extension that comes with visual studio when you first install it as additional options
